ohm-js/test/test-ohm-syntax.js

```diff
@@ -976,6 +976,47 @@ test('inheritance', t => {
   t.end();
 });
 
+test('override with "..."', t => {
+  let g = ohm.grammar('G { letter := "@" | ... }');
+  t.equal(g.match('@', 'letter').succeeded(), true);
+  t.equal(g.match('a', 'letter').succeeded(), true);
+
+  g = ohm.grammar('G { letter := ... | "@" }');
+  t.equal(g.match('@', 'letter').succeeded(), true);
+  t.equal(g.match('a', 'letter').succeeded(), true);
+
+  g = ohm.grammar('G { letter := "3" | ... | "@" }');
+  t.equal(g.match('@', 'letter').succeeded(), true);
+  t.equal(g.match('3', 'letter').succeeded(), true);
```

Comment:
Nit: we should add a test to make sure that `letter` matches `'a'`, too.

Reply:
Done.